scripts/exregional_run_met_ascii2nc_obs.sh

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env bash
+
+#
```

Comment:
A doc block here would also be helpful.

Reply:
Since no other tasks in verification have a doc block here, I'll decline to add if here for now, but something to add for the future.